### PR TITLE
Value sets for the Location profile

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling , Hi Matt, Kindly review the pull request.

### CODE REVIEW

File: input/vocabulary/valueset-service-delivery-location-role-type.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-service-delivery-location-role-type.json:1]: Note, we might need to repurpose this for Place of Service (POS), but TBD.

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
Changes required:
- Add Location.physicalType
- Add or link (from PractitionerRole) Location.hoursOfOperation.daysOfWeek

File: input/vocabulary/valueset-service-delivery-location-role-type.json
Comment (enjoysparkling) [reply input/vocabulary/valueset-service-delivery-location-role-type.json:1]: No action needed on this one. Will be addressed by #76.

**Comment by Gokulsignal** [comment]
Hi @enjoysparkling , 

Added Location.physicalType on the Location profile.

Kindly review.

**Review by enjoysparkling — APPROVED** [review]
Looks good to go.

**Pull request merged**